daemon/sdnotify.go

```diff
@@ -2,30 +2,33 @@
 package daemon
 
 import (
-	"errors"
 	"net"
 	"os"
 )
 
-var SdNotifyNoSocket = errors.New("No socket")
-
 // SdNotify sends a message to the init daemon. It is common to ignore the error.
```

Comment:
For reference, bool+error is also how rust-systemd bindings do it: https://github.com/jmesmon/rust-systemd/blob/master/src/daemon.rs#L189